main.cpp

```diff
@@ -743,6 +870,15 @@ void LFOSection::displayInitState()
               << "Trigger mode index = " << triggerModeIndex << std::endl;
 }
 
+void LFOSection::shrinkPan(int stereo)
+{
+    while (stereoSpreader > stereo)
+        {
+            stereoSpreader -= 5;
+            std::cout << "Stereo shrinked to " << stereoSpreader << std::endl;
+        }
```

Comment:
FIXME: indentation
curly braces should line up with the keywords on the line above

- [ ] revise